src/components/TransactionItemComponent/DataCeils/DateCellProps.ts

```diff
@@ -0,0 +1,9 @@
+import type Transaction from '@src/types/onyx/Transaction';
+
+type DataCellProps = {
```

Comment:
There is some naming confusion here:
file called `DateCellProps` property called DataCellProps.

In general I think the name DataCellProps is a bit generic (even though these components are supposed to be generic).
Maybe we can go with `TransactionDataCellProps` or something similar? Also the directory name is `DataCeils` - should be Cells